src/components/map/Layer.js

```diff
@@ -27,6 +27,8 @@ class Layer extends PureComponent {
         isVisible: true,
     };
 
+    state = {};
```

Comment:
Makes this null value check redundant? https://github.com/dhis2/maps-app/blob/616f5f27deb1658a62a27e012a701a024ccdc474/src/components/map/Layer.js#L48

Reply:
Yes, that check is no longer needed. Now remoived :-)